### PR TITLE
Improve PollOption Markup

### PR DESCRIPTION
**Pull request by DavideIadeluca**
**Fixes #0000**

**Changes proposed in this pull request:**
Change markup and thus improve accessibility/usability

**Reviewers should focus on:**
Let's wait with merging this into im/global-polls because there seem to be some issues with the UI already. 

**Screenshot**
UI is mostly the same. The only _visible_  changes are that all PollOptions now have the same width within one Poll and that the `cursor: pointer;` is now shown when hovering over the `PollBar` instead of just the checkbox, to reflect where the user can click to vote.

**Confirmed**

- [ ] Frontend changes: tested on a local Flarum installation.
- [ ] Backend changes: tests are green (run `composer test`).

### CODE REVIEW

File: js/src/forum/components/Poll/PollOption.tsx [hunk 1]
```diff
@@ -50,24 +45,39 @@ export default class PollOption extends Component<PollOptionAttrs, PollState> {
...
+          <div className="PollAnswer-checkbox">
+            <input
+              className="PollAnswer-input sr-only"
+              type="checkbox"
+              id={this.option.id()}
+              name={this.name}
+              value={this.answer}
+              checked={this.voted}
+              disabled={isDisabled}
+              aria-labelledby={`${this.name}-${this.option.id()}-label`}
```
Comment (novacuum) [inline js/src/forum/components/Poll/PollOption.tsx:69]: add the related id on the "label" element -> .PollAnswer-text-answer

File: js/src/forum/components/Poll/PollOption.tsx [hunk 2]
```diff
@@ -50,24 +45,39 @@ export default class PollOption extends Component<PollOptionAttrs, PollState> {
...
+              aria-describedby={`${this.name}-${this.option.id()}-description`}
```
Comment (novacuum) [inline js/src/forum/components/Poll/PollOption.tsx:70]: remove, or maybe change it to the id of the poll description element (if it exists)

**Review by novacuum — COMMENTED** [review]
I have commit some fixes: so that you can now test the full functionality

File: js/src/forum/components/Poll/PollOption.tsx (hunk 2 above)
Comment (DavideIadeluca) [reply js/src/forum/components/Poll/PollOption.tsx]: Removed

File: js/src/forum/components/Poll/PollOption.tsx (hunk 1 above)
Comment (DavideIadeluca) [reply js/src/forum/components/Poll/PollOption.tsx:69]: What is the related id?

**Comment by novacuum** [comment]
For me, all looks good.

**Review by imorland — APPROVED** [review]

**Pull request merged**